spec/acceptance/support/in_a_new_module.rb

```diff
@@ -1,7 +1,8 @@
 
 shared_context 'in a new module' do |name|
   before(:all) do
-    system("pdk new module #{name} --skip-interview") || raise
+    output = `pdk new module #{name} --skip-interview 2>&1`
+    raise "Failed to create test module:\n#{output}" unless File.directory?(name)
```

Comment:
Then you can also `raise` here on the exit status returned from capture2e, and not rely on environmental factors.